javascript/serializeEvent.js

```diff
@@ -28,9 +28,7 @@ function serializeEventDetails (event) {
   const details = {}
 
   for (const property of detailProperties) {
-    if (Object.prototype.hasOwnProperty.call(event, property)) {
-      details[property] = event[property]
-    }
+    details[property] = event[property]
```

Comment:
It might be worth checking if `event[property` is `undefined` here to avoid having an object like `{ a: undefined, b: undefined }` since not every event will have every property.